packages/create-block/lib/init-package-json.js

```diff
@@ -37,7 +37,9 @@ module.exports = async ( {
 	function checkDependency( packageArg ) {
 		const { type } = npmPackageArg( packageArg );
 		if (
-			! [ 'git', 'tag', 'version', 'range', 'remote' ].includes( type )
+			! [ 'git', 'tag', 'version', 'range', 'remote', 'alias' ].includes(
+				type
+			) // ✅ Add alias
```

Comment:
```suggestion
			! [ 'git', 'tag', 'version', 'range', 'remote', 'alias' ].includes(
				type
			)
```

I don't think we need such a comment.